partiql-value/src/lib.rs

```diff
@@ -367,8 +393,6 @@ impl NullableEq for Value {
 impl NullableOrd for Value {
     type Output = Self;
 
-    // TODO: comparison is not right for data type mismatches. Equality permits mistyped arguments
-    //  while comparison ops should return Missing
```

Comment:
This TODO was addressed in a previous PR: https://github.com/partiql/partiql-lang-rust/pull/222